src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -403,17 +403,27 @@ HRESULT STDMETHODCALLTYPE CorProfiler::JITCompilationStarted(
         continue;
       }
 
-      if (method_replacement.wrapper_method.method_signature.data.size() < 3) {
+      auto wrapper_method_signature_size =
+          method_replacement.wrapper_method.method_signature.data.size();
+
+      if (wrapper_method_signature_size < 3) {
```

Comment:
This makes me realize that we should actually have a minimum of 5 with the addition of the opCode and mdToken.

Reply:
Update this check to `< 5` in 0ba322610d681ddc0fc00a540665ae74c3b2ea10.